### PR TITLE
Create new ports and networks routing table

### PR DESCRIPTION
**Pull request by ianf77**
Inconsistencies in firewall documentation

https://issues.redhat.com/browse/AAP-21720

### CODE REVIEW

File: downstream/assemblies/platform/assembly-network-ports-protocols.adoc [hunk 1]
```diff
@@ -17,216 +17,75 @@ In some of the following use cases, hop nodes are used instead of a direct link
 .{PlatformNameShort} Network ports and protocols 
 image::aap-network-ports-protocols.png[Interaction of Ansible Automation Platform components on the network with information about the ports and protocols that are used.]
 
-The following tables show the default {PlatformName} destination ports required for each application.
+TThe following table indicates the destination port and the direction of network traffic:
```
Comment (EMcWhinn) [inline downstream/assemblies/platform/assembly-network-ports-protocols.adoc:20]:
```suggestion
The following table indicates the destination port and the direction of network traffic:
```

**Review by EMcWhinn — COMMENTED** [review]
Table looks good, just one typo in lead in sentence.

**Review by EMcWhinn — APPROVED** [review]
LGTM

**Comment by ariordan-redhat** [comment]
Backported to `2.4` in #1482

**Pull request merged**